canonical-web-and-design/ubuntu.com
# Add missing world map files for /desktop/statistics

## Done

- Added files that were mistakenly removed to make the world map work

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/statistics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the world map is visible


## Issue / Card

Fixes #8582

## Screenshots

![image](https://user-images.githubusercontent.com/441217/99909670-d81d6180-2ce1-11eb-8734-9c5d26fde3a5.png)
